space-plugins/nuxt-base/server/api/settings.ts

```diff
@@ -0,0 +1,78 @@
+export default defineEventHandler(async (event): Promise<any> => {
+	const { spaceId, accessToken } = event.context.appSession;
+	const appId = env('APP_ID');
```

Comment:
currently APP_ID needs to be hardcoded inside the .env as there is no way to retrieve it